src/lsp-server/Cargo.toml

```
@@ -19,6 +19,7 @@ mz-sql-parser = { path = "../sql-parser", default-features = false  }
 mz-sql-lexer = { path = "../sql-lexer", default-features = false }
 mz-ore = { path = "../ore", default-features = false, features = ["async", "test"] }
 regex = "1.7.0"
+mz-sql-pretty = { path = "../sql-pretty", default-features = false  }
```

```suggestion
mz-sql-pretty = { path = "../sql-pretty" }
```

Not sure that `default-features = false` is needed because this crate already includes `workspace-hack` and that's the only thing disabling the features does (it's for WASM compilation).